workflows/explore_eml/navigate_through_eml.Rmd

```diff
@@ -9,16 +9,23 @@ library(EML)
 cn <- CNode('PROD')
 adc <- dataone::getMNode(cn, 'urn:node:ARCTIC')
```

Comment:
Don't need `dataone::` since dataone is attached at the top.  Seems like we have inconsistent namespace use in the training, not a big deal though

Reply:
yeah...I decided not to deal with the inconsistent usage and instead added a note about it in the introduction